crates/pop-cli/src/commands/mod.rs

```diff
@@ -128,11 +128,11 @@ impl Command {
 					},
 				},
 			},
-			#[cfg(feature = "contract")]
 			Self::Test(args) => match args.command {
-				test::Command::Contract(cmd) => match cmd.execute().await {
-					Ok(feature) => Ok(json!(feature)),
-					Err(e) => Err(e),
+				None => test::Command::execute(args).await.map(|t| json!(t)),
+				Some(cmd) => match cmd {
+					#[cfg(feature = "contract")]
+					test::Command::Contract(cmd) => cmd.execute(&mut Cli).await.map(|t| json!(t)),
```

Comment:
```suggestion
					test::Command::Contract(cmd) => cmd.execute(&mut Cli).await.map(json!),
```

Not important but I realized you can do this as well :D